serviceinsight/license.md

```diff
@@ -12,6 +12,13 @@ related:
 - nservicebus/licensing
 ---
 
+There are a few options for adding a license to ServiceInsight.
+
+Note that the license file might allow running multiple products (NServiceBus Core, ServiceControl and ServiceInsight). Therefore the license file can also be loaded from other platform tools to achieve the same end result, as long as the licensing method imports the license into the registry.
```

Comment:
make this an include https://github.com/Particular/docs.particular.net#markdown-includes  since it is used twice